src/filesystem/files.rs

```diff
@@ -128,6 +128,15 @@ where
     pub fn flush(&mut self) -> Result<(), Error<D::Error>> {
         self.volume_mgr.flush_file(self.raw_file)
     }
+
+    /// Consume the `File` handle and close it. The behavior of this is similar
+    /// to using [`core::mem::drop`] or letting the `File` go out of scope,
+    /// except if the file fails to close properly, this returns an error,
+    /// whereas the automatic drop will panic. This method is therefore hihgly
```

Comment:
typo

Reply:
If you agreed that not panicking on drop was the way to go, I would remove the part about panicking here, and instead do something like:

```
/// Consume the `File` handle and close it. The behavior of this is similar
/// to using [`core::mem::drop`] or letting the `File` go out of scope,
/// except this lets the user handle any errors that may occur in the process,
/// whereas when using drop, any errors will be discarded silently.
```